.ci/common.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# Attention, there is no "-x" to avoid problems on Travis
+set -e
+
+DEBUG=true
+
+# result of command goes to $RUN_JOB
+# 0 = not run (skip)
+# 1 = run (no skip)
+function should_run_job {
+    local SKIP_JOB_BY_FILES=$1
+    local SKIP_FILES=$2
+    local OUT=""
+
+    echo "Initial value of SKIP_JOB_BY_FILES: $SKIP_JOB_BY_FILES"
+    echo "Files that are allowed to be skipped: $SKIP_FILES"
+
+    if [[ $SKIP_JOB_BY_FILES != 'false' ]]; then
+         ## Identify previous commit to know how much to check
+         ## Check assumes we are only working with 1 commit if we are in master
+         local PREVIOUS_COMMIT="HEAD~1"
+
+         if [[ $DEBUG == "true" ]]; then
+              OUT=`git log -10 --format="%h %B"`
+              echo "Current Branch log: $OUT"
+              OUT=`git log origin/master -10 --format="%h %B"`
+              echo "origin/master log: $OUT"
+         fi
+
+         ## Travis merges the PR commit into origin/master
+         ## This identifies the PR's original commit
+         ## if it notices a merge commit
+         local HEAD="HEAD"
+
+         if git show --summary HEAD | grep ^Merge: ; then
+              HEAD=`git log -n 1 --no-merges --pretty=format:"%h"`
+         fi
+
+         if [[ $DEBUG == "true" ]]; then
+              echo "Head commit: $HEAD"
+              OUT=$(git branch origin/master --contains $HEAD)
+              echo "Master contains head commit: $OUT"
+         fi
+
+         if [[ $(git branch origin/master --contains $HEAD \
+                    | wc -c ) == 0 ]]; then
+              PREVIOUS_COMMIT=`git merge-base origin/master $HEAD`
+         fi
+
+         echo "Previous Commit to start with: $PREVIOUS_COMMIT"
+
+         if [[ $DEBUG == "true" ]]; then
+              OUT=`git diff --name-only $HEAD $PREVIOUS_COMMIT`
+              echo "Files between top commit and previous: $OUT"
```

Comment:
simply:
```
echo "Files between top commit and previous:"
git diff --name-only $HEAD $PREVIOUS_COMMIT
```

please update all other places in the same way.

Reply:
Done.

Reply:
undone because of https://github.com/checkstyle/checkstyle/pull/7125#issuecomment-538569439